exonum/src/blockchain/schema.rs

```diff
@@ -64,46 +88,51 @@ where
 
     /// Returns table that represents a map from transaction hash into raw transaction message.
     pub fn transactions(&self) -> MapIndex<&T, Hash, RawMessage> {
-        MapIndex::new("core.transactions", &self.view)
+        MapIndex::new(TRANSACTIONS, &self.view)
+    }
+
+    /// Returns table that represents a map from transaction hash into execution status.
```

Comment:
Nitpick: using a consistent terminology (i.e., s/status/result/) could be better. This concerns naming of variables / methods as well.

Reply:
Actually, it isn't a nitpick: it is the result of several renamings and it should be fixed. :+1: